src/c++/perf_analyzer/docs/cli.md

```diff
@@ -0,0 +1,710 @@
+ * Neither the name of NVIDIA CORPORATION nor the names of its
+   contributors may be used to endorse or promote products derived
+   from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Perf Analyzer CLI
+
+## General Options
+
+### `-?`
+### `-h`
+### `--help`
+
+Prints a description of the Perf Analyzer command line interface.
+
+<br>
+
+### `-m`
+
+Specifies the model name for Perf Analyzer to run.
+
+This is a required option.
+
+<br>
+
+### `-x`
+
+Specifies the version of the model to be used. If not specified the most
+recent version (the highest numbered version) of the model will be used.
+
+<br>
+
+### `--service-kind=[triton|triton_c_api|tfserving|torchserve]`
+
+Specifies the kind of service for Perf Analyzer to generate load for. Note: in
+order to use `torchserve` backend, the `--input-data` option must point to a
+JSON file holding data in the following format:
+
+```JSON
+{
+  "data": [
+    {
+      "TORCHSERVE_INPUT": [
+        "<complete path to the content file>"
+      ]
+    },
+    {...},
+    ...
+  ]
+}
+```
+
+The type of file here will depend on the model. In order to use `triton_c_api`
+you must specify the Triton server install path and the model repository path
+via the `--triton-server-directory` and `--model-repository` options.
+
+Default is `triton`.
+
+<br>
+
+### `--model-signature-name=<string>`
+
+Specifies the signature name of the saved model to use.
+
+Default is `serving_default`. This option will be ignored if `--service-kind`
+is not `tfserving`.
+
+<br>
+
+### `-v`
+
+Enables verbose mode. May be specified an additional time (`-v -v`) to enable
+extra verbose mode.
+
+<br>
+
+## Measurement Options
+
+### `--measurement-mode=[time_windows|count_windows]`
+
+Specifies the mode used for stabilizing measurements. 'time_windows' will
+create windows such that the duration of each window is equal to
+`--measurement-interval`. 'count_windows' will create windows such that there
+are at least `--measurement-request-count` requests in each window and that
+the window is at least one second in duration (adding more requests if
+necessary).
+
+Default is `time_windows`.
+
+<br>
+
+### `-p <n>`
+### `--measurement-interval=<n>`
```

Comment:
Why two headers vs one header that is comma separated?

Reply:
No reason, I was just copying what git docs do:

https://git-scm.com/docs/git-fetch